common/models/role.js

```diff
@@ -171,13 +171,6 @@ module.exports = function(Role) {
   Role.isOwner = function isOwner(modelClass, modelId, userId, callback) {
     assert(modelClass, 'Model class is required');
     debug('isOwner(): %s %s userId: %s', modelClass && modelClass.modelName, modelId, userId);
-    // No userId is present
-    if (!userId) {
```

Comment:
I think this block should be preserved. If there is no `userId` provided by the request context, then there is no point in running any checks - they must all fail.